doc/PRE_UPGRADE.d/1.3.0.md

```diff
@@ -0,0 +1 @@
+We should encourage people to have a look at the Changelog: https://fossil.kd2.org/paheko/wiki?name=Changelog/1.3 when upgrading to this version because there are major changes. If you have installed some plugins, not all will be compatibles.
```

Comment:
```suggestion
We encourage you to have a look at the Changelog: https://fossil.kd2.org/paheko/wiki?name=Changelog/1.3 when upgrading to this version because there are major changes. If you have installed some plugins, not all will be compatibles.
```